pkg/rsyslog/rsyslog.conf

```diff
@@ -18,7 +18,8 @@ $FileCreateMode 0640
 $DirCreateMode 0755
 $Umask 0022
 
-$outchannel local,/persist/rsyslog/syslog.txt,10485760,/rotate.sh
+$outchannel local,/persist/rsyslog/syslog.txt,10485760,/rotate.sh syslog
+$outchannel logmanager,/persist/log/logmanager.log,104857600,/rotate.sh logmanager
```

Comment:
Why keep this in /persist/log/ as opposed to /persist/rsyslog/logmanager.log?

Reply:
No specific reason. Will move to /persist/rsyslog

Reply:
Can you please drop the size of the file back to 10Mb